entrypoint.sh

```diff
@@ -7,6 +7,11 @@ log() {
 }
 log "Starting Kea ${KEA_EXECUTABLE} container"
 
+# Make sure there is no leftover from previous process if it was abruptly aborted (power shutdown for instance).
+# Kea does not restart if the pid file from the previous process still exists. This ensures that it can restart
+# without any issue.
+rm -f /usr/local/var/run/kea/dhcp4.kea-dhcp4.pid
```

Comment:
This works for the DHCPv4 instance, but none of the others. You should probably make something similar to 
```
rm -f /usr/local/var/run/kea/*.pid
```
to get a better chance of removing all possible files.

It was not trivial to find, but [here](https://github.com/isc-projects/kea/blob/3e7dcb311994b0b2538cf4c4d82339bfbb6b7b2a/doc/sphinx/arm/ddns.rst#starting-and-stopping-the-dhcp-ddns-server) we can read the following:

-----------------------------------------------
During startup, the server attempts to create a PID file of the form: [runstatedir]/[conf name].kea-dhcp-ddns.pid where:

- `runstatedir` - is the value as passed into the build configure script; it defaults to "/usr/local/var/run". Note that this value may be overridden at runtime by setting the environment variable KEA_PIDFILE_DIR. This is intended primarily for testing purposes.
- `conf name` - is the configuration file name used to start the server, minus all preceding paths and the file extension. For example, given a pathname of "/usr/local/etc/kea/myconf.txt", the portion used would be "myconf".

If the file already exists and contains the PID of a live process, the server issues a DHCP_DDNS_ALREADY_RUNNING log message and exits. It is possible, though unlikely, that the file is a remnant of a system crash and the process to which the PID belongs is unrelated to Kea. In such a case it is necessary to manually delete the PID file.

-----------------------------------------------

So while it might be overkill, we should probably make something like 

```
if [ -n "${KEA_PIDFILE_DIR}" ]; then
    rm -f "${KEA_PIDFILE_DIR}"/*.pid
else 
  rm -f /usr/local/var/run/kea/*.pid
fi
```

Furthermore, there is nothing stopping a user from sharing a mount of this location between instances, and then we will remove other container's pid files. And while I am not 100% certain this is the pattern at least looks like we can narrow a particular container's PID file to something like this:

```
rm -f /usr/local/var/run/kea/*.kea-"${KEA_EXECUTABLE}".pid
```

But thinking about this extreme case we need a user to actually care to mount this particular folder (for some weird reason) so perhaps this last part is not necessary.